core/src/main/kotlin/org/evomaster/core/problem/graphQl/FullType.kt

```diff
@@ -0,0 +1,17 @@
+class FullType {
```

Comment:
for these classes, add a bit of documentation inside `/**  */` comments, also for the fields

Reply:
done